mlir/lib/Catalyst/Transforms/disable_assertion.cpp

```diff
@@ -0,0 +1,55 @@
+// Copyright 2023 Xanadu Quantum Technologies Inc.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+
+//     http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#define DEBUG_TYPE "assertion"
+
+#include "llvm/Support/Debug.h"
+
+#include "mlir/Dialect/LLVMIR/LLVMDialect.h"
+#include "mlir/Pass/Pass.h"
+#include "mlir/Transforms/GreedyPatternRewriteDriver.h"
+
+#include "Catalyst/IR/CatalystDialect.h"
+#include "Catalyst/Transforms/Patterns.h"
+
+using namespace llvm;
+using namespace mlir;
+using namespace catalyst;
+
+namespace catalyst {
+#define GEN_PASS_DEF_DISABLEASSERTIONPASS
+#include "Catalyst/Transforms/Passes.h.inc"
+
+struct DisableAssertionPass : impl::DisableAssertionPassBase<DisableAssertionPass> {
+    using DisableAssertionPassBase::DisableAssertionPassBase;
+
+    void runOnOperation() final
+    {
+        LLVM_DEBUG(dbgs() << "disable assertion pass"
+                          << "\n");
+
```

Comment:
I realize we have some custom debug messages in some of the other passes, but to be honest MLIR already has built-in debug information that can show you what passes are being run, so I would probably remove this from here since it doesn't provide any additional info beyond that the pass is being run.
```suggestion
```

Reply:
Updated in new commit